public/usage-examples/graphics/fill_circle_on_bitmap-1-beyond.cpp

```diff
@@ -0,0 +1,115 @@
+#ifdef APPLE
+#include <SDL.h>
+#else
+#include <SDL2/SDL.h>
+#endif
+#include <iostream>
```

Comment:
```suggestion
#include <iostream>
#ifdef APPLE
#include <SDL.h>
#else
#include <SDL2/SDL.h>
#endif
```

Ideally imports should be in this order. While it doesn't cause any issues but this is the ideal practice and is followed over other examples.